docs/docs/spark-writes.md

```diff
@@ -228,6 +232,24 @@ SET spark.wap.branch = audit-branch
 INSERT INTO prod.db.table VALUES (3, 'c');
 ```
 
+### Via DataFrames
+
+Branch writes via DataFrames can be performed by providing a branch identifier, `branch_yourBranch` in the operation.
+
+To insert into `audit` branch
```

Comment:
How about make these comments, to match the sql section?

Reply:
I could not find relevant examples in code to match the SQL section, (merge into branch and `SET spark.wap.branch` option) so I included the basic examples of insert and update (overwritePartitions). Let me know if we need to add more examples.

Reply:
Sorry I meant, can we make them code comments ie

```// To insert into audit branch```

because they are comments in SQL section?

Reply:
Ah, I misunderstood. Done.